manifest.dev.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
what all envs are requiring a separate manifest?

Reply:
The main split we want right now is a separate "playtest" version (with a unique name and add-on ID and a control panel), for the production version we just want it to collect data and silently submit it.

The separate name and addon ID are so they can both be installed at the same time and it's possible to tell them apart :)